### ISSUE TITLE
Consider recording meetings

### ISSUE THREAD

**Issue by eriktaubeneck** [patcg/meetings#10]
This is a branch of of issue #5 to discuss if we want to record the PATCG meetings.

From @michaelkleber comment:
> The W3C Process is that meetings can only be recorded if everyone involved agrees.

**Comment by drpaulfarrow** [comment]
Normal procedure wrt recording is: the host of the meeting asks at the onset of recording if it is ok with everyone that the session is recorded. If there are any objections, then no recording. Could we do it that way?

**Comment by eriktaubeneck** [comment]
I'm not personally opposed to recordings of meetings, but I am in favor of making the meetings as inclusive as possible. Given that there seem to be some in #5 opposed to recording, I'd favor not recording so that those folks (and presumably others) are able and willing to participate.

**Comment by JoelPM** [comment]
Having read meeting notes and also scribed them, it can be challenging to accurately capture the discussion. It also more-or-less precludes participation for the scribe due to the need to stay focused on scribing (yes, you can switch scribes to ask a question, but it requires coordination).

Would it be possible to capture an audio-only recording for the purpose of transcription after-the-fact?

**Comment by drpaulfarrow** [comment]
Recording the meeting will make it more inclusive in total, as folks who can't attend will still feel they can participate in some way. And if people joining the meetings really do not want to be recorded, they can signal that at the beginning and recording will not happen. And yes, audio recording would also be fine.

**Comment by eriktaubeneck** [comment]
> Recording the meeting will make it more inclusive in total
I would disagree here. I think the difference between reading a transcript and watching/listening to a recording, while not exactly the same, a much smaller delta than preventing people who would otherwise contribute but are uncomfortable (or even prevented from) participating due to recording.

> And if people joining the meetings really do not want to be recorded, they can signal that at the beginning and recording will not happen.
I'm highly confident that this will result in no meetings being recorded (as is the case with other related groups), and I would argue it's better to reach explicit consensus before the meeting.

**Comment by anderagakura** [comment]
Recording only audio would be a good consensus but at the beginning of the meeting, when we'll hear "recording is on" we might see some people leaving the room.

Recording the meeting would be great to follow the topics. But I'm reading some concerns about it (discouraging, not fan etc...) so we probably should not do it. In this case, the notes have to be crystal clear to understand the topics and the context behind most of the discussions

**Comment by lmarkou1** [comment]
If there are confidentiality reasons for the meetings to not be recorded, then potentially a second meeting could be set up or we turn off recording for a portion. But I assume nothing that confidential should be shared anyway. If simply because people don't like to hear themselves, hoping we can move past for the greater good. Other reasons for not recording?

**Comment by stumakha** [comment]
> Having read meeting notes and also scribed them, it can be challenging to accurately capture the discussion. It also more-or-less precludes participation for the scribe due to the need to stay focused on scribing (yes, you can switch scribes to ask a question, but it requires coordination).
> 
> Would it be possible to capture an audio-only recording for the purpose of transcription after-the-fact?

+1 from the point of view of someone who ever tried to be a scribe. 
Couple of other options:
a.  organizer and/or a scribe only have access to voice recording for a limited time (1 week?) and confirm deletion (snapshot?) after the notes are cleaned up. 
b. Use auto-transcribe feature only, without recording. With acronyms and accents it is rarely great but better than nothing and leaves room for "plausible deniability" for the lack of better term.

**Comment by chrisn** [comment]
Here's the W3C process guidance on recordings: https://www.w3.org/2021/Process-20211102/#meeting-recording

**Comment by bmayd** [comment]
Along with concerns mentioned above I'd add that I have never seen scribes capture things consistently well: at best they manage a terse summary which misses a lot and more generally what ends up being written bares little or no resemblance to what was actually said and misses the point. At times I have tried to correct what's been captured of what I've said, but generally it is so off I don't bother, preferring to follow the continuing conversation, and I'm sure I'm not alone in that. None of this is the fault of scribes, who I am very thankful to for the effort, it just seems we're asking more from folks than they can deliver.

I've also noted in all the meetings I attend apart from one with a dedicated scribe, the effort to get someone to volunteer is becoming increasingly cumbersome and protracted; its understandable because, as Joel noted, it means you're not able to participate, which defeats the purpose for attending the meeting.

If we could figure out a way of using an automated transcription, I think it would be a service to all concerned. Failing that, a dedicated scribe would be a less preferred option, but better than having to ask folks who want to contribute to sacrifice their participation.

**Comment by brodrigu** [comment]
To those that are unable to attend meetings, a full recording, an audio-only recording, an automated transcript, and finally a manual transcript are likely the ordered set of preferred alternatives that provide the most engagement and inclusion aside from an accommodating meeting time.

It's clear that we are not going to be able to find a time/date for meetings that will accommodate all participants and it will likely be a time that is friendly to participants in the United States and either Europe or APAC (but not both).

For those privileged enough to be accommodated by the meeting time, a recording is unnecessary. However, even the time-accommodated party ,who because of their preference for not being recorded chooses not to participate in the meeting, has the option to either listen passively to the live meeting, or if recorded, listen later at their leisure. They are afforded the same ability to participate as those in an un-accomodated timezone.

I would encourage the group to consider full-recording (video + audio) the meeting by default, with perhaps some option for those that don't wish to be recorded (or cannot attend in person) to provide their contributions in an accommodating way.

Example options include:
* contributors can ask for recording to be paused during their segment of participation
* a section of the meeting can be reserved for scribe-only presentations / feedback
* questions or feedback can be provided via text/chat with the meeting chairs reading aloud into the meeting
* asynchronous feedback could be presented at the beginning / end of the meeting to accommodate those who listened to the recording between meetings and have something to add

We owe it to those interested parties outside of the privileged timezones to consider the level of participation we can offer them against the preferences of those that are able to attend the chosen time.

**Comment by npdoty** [comment]
> If there are confidentiality reasons for the meetings to not be recorded, then potentially a second meeting could be set up or we turn off recording for a portion. But I assume nothing that confidential should be shared anyway. If simply because people don't like to hear themselves, hoping we can move past for the greater good. Other reasons for not recording?

Just in case people are interested in the reasons and to quickly summarize for this issue thread, I believe recording discourages candor in participation, and discourages some people from participating at all, and provides an incomplete record if we plan to follow W3C practices and won't coerce people who do participate but don't want to be recorded. These are, incidentally, instances where privacy is valuable because it prevents chilling of speech.

I'm not aware of W3C meetings where we have had trouble getting volunteers to scribe in real time but did have people volunteering to go back and transcribe an audio recording after the fact (without anyone able to help make corrections). It sounds laborious to me, but if there is willingness to do that, it's possible (though not certain) that we could get unanimous agreement for recordings that would be used only for that purpose.

**Comment by ekr** [comment]
I don't see a lot of point in continuing to discuss this, given that the process appears to preclude recording if anyone objects and people clearly object.

With that said, IETF records WG meetings as a matter of policy, and I haven't observed it leading to significantly less candor than in W3C meetings.

**Comment by chris-wood** [comment]
Strong +1 to @ekr. I think the process needs to be transparent if it's going to succeed.

**Comment by martinthomson** [comment]
The W3C policy is not a good fit with their mission.  We make decisions at these meetings that affect a great many people and conducting them essentially in private (or through the filter of an amateur note-taker) puts those who are unable to attend at a significant disadvantage.  Denying people access to a recording is an exercise of privilege that is neither justified nor necessary.

However, it is the established policy.  If people want to use that policy to block recording, I won't fight that.  So what @ekr said.

**Comment by appascoe** [comment]
It seems like I'm generally against the grain here. Perhaps I should clarify my position more fully.

From the personal side, I actively try to limit my online presence. Sometimes this is not strictly possible due to work obligations. Most of this has to do with how I value my privacy; despite being in adtech, I am an advocate for increased privacy protections. Having my likeness or voice available in perpetuity publicly is... less than desirable.

From the standards process side, there are benefits, as Nick said, to increasing the level of candor in the discussion. The topics in particular for this group are in the sights of general media attention. The standards we are evaluating can also be highly contentious due to perceived threats to privacy as well as perceived threats to extant business models. (To wit: the entire #NoFLoC campaign, in my opinion, largely missed the purpose of FLoC and misinterpreted its privacy measures.) While some argue that this should be reason to increase transparency, I don't see why this should extend to literal recordings of said exchanges. The end result of the standards process for this group will be published, available for review and additional comment, and that is largely what matters. Drafts will be available on Github. If a participant wishes to publicize their contributions or the effects of standards as they're being developed, they are free to do so of their own accord. privacysandbox.com is a prime example.

Third, there are complaints about the fidelity of the work of scribes. While I agree that these can vary in quality, I actually used one today. Michael sent out the minutes from the January 5 FLEDGE meeting, which I was unable to attend as I was on PTO. I read the notes in five minutes and, as far as I can tell, largely captured the main points of the discussion. I'm glad I did not have to wade through a one-hour recording. If I required clarification, I could comment on the linked issues in question.

Finally, there's the discussion of fairness and disadvantage. First, we've agreed to rotate the timezones for the meetings. I am aware that this may preclude my attendance at a future date. That's a loss, and I don't think a video or audio recording will significantly recover that loss. The advantage of attendance is participation in the discussion, the ability to ask questions in the moment, and proffer one's own ideas to receive immediate feedback. Recordings are no substitute for this. Moreover, the bulk of the discussions should be happening asynchronously on Github anyway. If they're not, to the point where real progress is only made during a couple three-hour marathon meetings once a quarter or so, I think there's a larger problem in the process.

**Comment by michaelkleber** [comment]
I just read through the lengthy https://github.com/w3c/w3process/issues/334, the most recent (AFAIK) discussion of the W3C's process regarding the recording of meetings.

First, let me correct a mistake I made earlier.  Over in #5 and quoted at the top of this issue, I said "The W3C Process is that meetings can only be recorded if everyone involved agrees," and @chrisn linked to the [relevant section](https://www.w3.org/2021/Process-20211102/#meeting-recording) of the Process Doc.  I now see that, as explained [here](https://github.com/w3c/w3process/issues/334#issuecomment-665410385), that Process Doc governs Working Groups — but _Community_ Groups like PATCG are designed to be more flexible.  (It still seems clear that recordings cannot happen without consent, though.)

Second, I recommend looking at [this comment](https://github.com/w3c/w3process/issues/334#issuecomment-662659706) and the others that refer back to it.  It seems clear that recordings that get published are a serious risk of being used for harassment, in ways that I as a white guy have not needed to worry about.  This CG is likely to work on proposals that trigger strong reactions from some parts of the web, which I expect further elevates this risk.  This consideration was enough to change my mind quite convincingly.

**Comment by bmayd** [comment]
I mentioned the question of automated transcriptions to someone who pointed me at (https://w3c.github.io/Guide/meetings/transcripts) which reads in part:

>W3C’s policy is that automated transcriptions of W3C meetings are solely for the use of those present in the meeting and must not be saved or shared with anyone not present in the meeting. 

(apparently me and Michaeal were writing at the same time)

**Comment by michaelkleber** [comment]
To be clear, I believe the harassment concern is about published recordings, not about tech used to get better written notes.

**Comment by eriktaubeneck** [comment]
> Second, I recommend looking at [this comment](https://github.com/w3c/w3process/issues/334#issuecomment-662659706) and the others that refer back to it. It seems clear that recordings that get published are a serious risk of being used for harassment, in ways that I as a white guy have not needed to worry about. This CG is likely to work on proposals that trigger strong reactions from some parts of the web, which I expect further elevates this risk. This consideration was enough to change my mind quite convincingly.

Thank you for sharing @michaelkleber. Accessibility and inclusion are incumbent to our mission here (and that of the W3C), and it seems clear that there are both positives and negatives along this dimension when it comes publicly available recordings. I also believe that we can capture those positives in other ways. For example, 
1. We've committed to rotating the timing of the meetings, discussed in #4, so that every 3rd meeting will be inconvenient or inaccessible to each timezone. 
2. We are discussing in #6 the best approach to take useful minutes and notes, and I have committed to contribute to the scribing effort when I'm able to attend.

I am confident we will be able to find ways to minimize the legitimate concerns of those who would find a recording helpful, without creating the risk that the linked comment clearly articulates. If there are other specific concerns (and hopefully creative solutions), I'd encourage folks to raise issues specific to those concerns so that we can address them (similar to how they are addressed in the two issues above.)
